.env

```diff
@@ -1,2 +1,3 @@
 export APP_SETTINGS="config.DevelopmentConfig"
 export DATABASE_URL="postgresql://localhost/language_learner_dev"
+export UNSPLASH_ACCESS_KEY="62365b47db82a7cf5c2557d8f6b2d2b5627c092b4304a12fdc70dccfd77547e1"
```

Comment:
Should this be visible in the repo?

Reply:
Not sure why this file is being tracked; i'll check